src/extension.ts

```diff
@@ -62,6 +62,28 @@ export function activate(context: ExtensionContext) {
     }
 }
 
+function reportIssue(gaugeVersion: cp.SpawnSyncReturns<string>) {
+    let extVersion = extensions.getExtension("getgauge.gauge").packageJSON.version;
+    let issueTempate = `\`\`\`
+VS-Code version: ${vscode.version}
+Gauge Extension Version: ${extVersion}
+${gaugeVersion.stdout.toString()}
+\`\`\``;
+
+    return opn(`https://github.com/getgauge/gauge-vscode/issues/new?body=${issueTempate}`).then(() => { }, (err) => {
+        vscode.window.showErrorMessage("Can't open issue URL. " + err);
+    });
+}
+
+function showReferences(languageClient: LanguageClient): any {
```

Comment:
showReferences function has been replaced with old code. Check master branch for correct code